k8s/operators/src/diskpool/main.rs

```diff
@@ -647,7 +672,26 @@ async fn ensure_crd(k8s: Client) {
             }
         }
     } else {
-        info!("CRD present")
+        info!(
+            "Creating CRD: {}",
+            serde_json::to_string_pretty(&crd).unwrap()
+        );
+
+        let pp = PostParams::default();
+        match dsp.create(&pp, &crd).await {
+            Ok(o) => {
+                info!(crd = ?o.name_any(), "created");
+                // let the CRD settle this purely to avoid errors messages in the console
```

Comment:
this piece of code is duplicated, please reuse.

Reply:
Incorporated